src/desktop/modules/controllers/batchDetails.ts

```diff
@@ -42,7 +43,7 @@ export default class BatchDetails {
 
       if (_.get(response, 'docs.length')) {
         batchData = response.docs[0];
-        res.status(200).send(Response.success(API_ID, { response: batchData }, req));
+        res.status(200).send(xss(JSON.stringify(Response.success(API_ID, { response: batchData }, req))));
```

Comment:
@sedin-tushar why u have added this code? please add the description

Reply:
should i add Description in each file ? there are about 10 files with this change